ide-eclipse.html

```diff
@@ -0,0 +1,366 @@
+<h2>JavaFX <span class="JFX">11</span> and Eclipse</h2>
+
+<p>This section explains how to use Java <span class="JDK">11</span> and JavaFX <span class="JFX">11</span> from Eclipse. 
+    Version Eclipse 2018-09 (4.9) Build id: I20180906-0745 was used for the following screenshots.
+</p>
+
+<p>
+    Download an appropriate <a href="https://jdk.java.net/11/">JDK <span class="JDK">11</span></a> for your operating system. 
+    Make sure <kbd>JAVA_HOME</kbd> is properly set to the Java <span class="JDK">11</span> installation directory. 
+</p>
+
+<p>
+    Include the new JDK as <kbd>Installed JREs</kbd> in <kbd>Eclipse -> Preferences -> Java -> Installed JREs -> Add</kbd>.
+</p>
+
+<div class="alert alert-warning">
+    <strong>Note: </strong>
+    If you get this message when you add JDK <span class="JDK">11</span>
+
+<pre><code>
+You selected a JRE that this version of Eclipse JDT does not yet support fully. Some of the features may not work as expected.
+</code></pre>
+    
+then install the patch from the Eclipse MarketPlace: 
+<a href="https://marketplace.eclipse.org/content/java-11-support-eclipse-2018-09-49">Java 11 support for Eclipse 2018-09 (4.9)</a>.
+</div>
+
+<p>
+    You can create a JavaFX <span class="JFX">11</span> project and use <a href="#IDE">IDE</a>,
+    <a href="#Maven">Maven</a> or <a href="#Gradle">Gradle</a> build tools.
+</p>
+
+<a name="IDE"></a><h3>IDE</h3>
+
+<p>
+    Follow these steps to create a JavaFX project and use the IDE tools to build it and run it.
+</p>
+
+<p>
+    Download the appropriate
+    <a target="_blank" href="https://gluonhq.com/products/javafx/">JavaFX SDK</a>
+    for your operating system and unzip it to a desired location, for instance
+    <kbd>/Users/your-user/Downloads/javafx-sdk-11</kbd>.
+</p>
+
+<p>
+    Create a new <kbd>User Library</kbd> under <kbd>Eclipse -> Window -> Preferences -> Java -> Build Path -> User Libraries -> New</kbd>. 
+    
+    <a href="images/ide/eclipse/ide/eclipse01.png" target="_blank"><img src="images/ide/eclipse/ide/eclipse01.png" alt="Add User Library"/></a>
+   
+    Name it <kbd>JavaFX11</kbd> and include the jars under the <kbd>lib</kbd> folder from JavaFX <span class="JFX">11</span>.
+    
+    <a href="images/ide/eclipse/ide/eclipse02.png" target="_blank"><img src="images/ide/eclipse/ide/eclipse02.png" alt="Add JavaFX jars"/></a>
+   
+</p>
+
+<p>
+    <strong>1. Create a Java project</strong>
```

Comment:
Can we use `<h5>` instead of <strong>?

Here is an example of how it looks after its added to `gluon.css` header css:

![image](https://user-images.githubusercontent.com/3197675/48185347-b51df000-e35a-11e8-8c8a-0301e5785c05.png)

Reply:
done